src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -162,22 +162,26 @@ class SidebarPanels extends React.Component<Props> {
                         exact
                         path={[
                             `/${SIDEBAR_VIEW_ACTIVITY}`,
+                            `/${SIDEBAR_VIEW_ACTIVITY}/annotations/:fileVersionId/:annotationId`,
                             `/${SIDEBAR_VIEW_ACTIVITY}/:activeFeedEntryType(annotations|comments|tasks)/:activeFeedEntryId?`,
```

Comment:
Should annotations still be in this list? Could we do something like this to avoid the need for the annotation-specific logic in the render function below?

```
path={[
    `/${SIDEBAR_VIEW_ACTIVITY}`,
    `/${SIDEBAR_VIEW_ACTIVITY}/:activeFeedEntryType(annotations)/:fileVersionId/:activeFeedEntryId?`,                        
    `/${SIDEBAR_VIEW_ACTIVITY}/:activeFeedEntryType(comments|tasks)/:activeFeedEntryId?`,
]}